omocha-domain/src/main/java/org/omocha/domain/auction/AuctionServiceImpl.java

```diff
@@ -77,4 +79,17 @@ public void removeAuction(AuctionCommand.RemoveAuction removeCommand) {
 		auctionReader.removeAuction(auction);
 	}
 
+	@Override
+	@Transactional(readOnly = true)
+	public Page<AuctionInfo.RetrieveMyAuctions> retrieveMyAuctions(
+		AuctionCommand.RetrieveMyAuctions retrieveMyAuctionsCommand, Pageable pageable) {
+
+		// TODO : AuctionStatus 에 따라 필요한 값이 다름
+		// 	biidng - nowPrice , conclude - concludePrice
+		return auctionReader
+			.getMyAuctionList(retrieveMyAuctionsCommand.memberId(), retrieveMyAuctionsCommand.auctionStatus(),
+				pageable);
```

Comment:
```suggestion
		return auctionReader.getMyAuctionList(
			retrieveMyAuctionsCommand.memberId(),
			retrieveMyAuctionsCommand.auctionStatus(),
			pageable
		);
```

이런식으로 변경하면 좋을 것 같아요!

Reply:
변경하였습니다.